### PR TITLE
feature/folder and loading files created and working

### PR DESCRIPTION
**Pull request by sergiohortua**

### CODE REVIEW

File: src/components/Loading/index.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Spinner } from "./style";
+
+// eslint-disable-next-line import/prefer-default-export
+export const Loading2 = () => {
```
Comment (alfredomtzg) [inline src/components/Loading/index.jsx]: Can be use export default const Loading2 and change the name to Loading

File: src/containers/App.jsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 import { GlobalStyle } from "../components/styles/globals";
-import Landing from "../pages/Landing";
+import {Loading2} from '../components/Loading/index'
 
 function App() {
   return (
     <div className="App">
       <GlobalStyle />
-      <Landing />
+      <Loading2/>
```
Comment (alfredomtzg) [inline src/containers/App.jsx]: delete this component in App.jsx

File: src/containers/App.jsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import React from "react";
 import { GlobalStyle } from "../components/styles/globals";
-import Landing from "../pages/Landing";
+import {Loading2} from '../components/Loading/index'
```
Comment (alfredomtzg) [inline src/containers/App.jsx:4]: delete this import because it is likely to conflict with develop branch

File: src/containers/App.jsx [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
```
Comment (alfredomtzg) [inline src/containers/App.jsx]: delete this import because it is likely to conflict with develop branch

**Review by alfredomtzg — CHANGES_REQUESTED** [review]
check the comments

**Review by alfredomtzg — APPROVED** [review]

**Review by wfercanas — APPROVED** [review]

**Pull request merged**